package.json

```diff
@@ -13,7 +13,7 @@
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
     "lint": "markdownlint docs/",
-    "lint-mdx": "eslint docs/ --ext mdx",
+    "lint-mdx": "eslint docs/ --ext mdx --fix",
```

Comment:
I think package.json should contain two commands:

1. that auto formats code so that contributors can run it locally. This command can be documented in README or configured to auto-run via husky.
2. a command that fails if eslint identifies any lint issues. This command should run in a Github workflow.

Reply:
great feedback here. would you like to implement this @gulshanpr ?

Reply:
Yeah, thanks @rootulp I'm working on it

Reply:
awesome!